src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -1030,7 +1030,7 @@ def test_edit_media_preview_add_two(self):
 
 
 class BaseTestEditDetails(BaseTestEdit):
-    __test__ = True
```

Comment:
Not a huge deal but we were running those tests twice since the classes that are inheriting from it were also running them. It's a base class, it should not be run.